js/widgets/toolbar.js

```diff
@@ -47,7 +48,7 @@ return $.widget( "mobile.toolbar", {
 
 	_create: function() {
 		var leftbutton, rightbutton,
-			role = this.element.is( ":jqmData(role='header')" ) ? "header" : "footer",
+			role =  this.element.is( ":jqmData(role='header')" ) ? "header" : "footer",
```

Comment:
@arschmitz I'm not exactly sure what you want to see here any longer
- do we want to keep the init selector as `data-role='header'` or `data-role='footer'` with the addition of the `data-toolbar-role='header'/'footer'` (which seems really redundant) OR do we want to change it to `data-role='toolbar'` with the `data-toolbar-role='header'/'footer'` where `header` is the default